doc/nrf/templates/library_template_README.rst

```diff
@@ -0,0 +1,137 @@
+     Use the provided stock phrases and includes when possible.
+   * Sections with ``*`` are optional and can be left out.
+     All the other sections are required for all libraries.
+     Do not add new sections or remove a mandatory section without consulting a tech writer.
+     If any new section is needed, try first to fit it as a subsection of one of the sections listed below.
+   * Include a tech writer in the PR review.
+
+Overview
+********
+
+.. note::
+   Use this section to give a general overview of the library.
+
+Implementation
+==============
+
+.. note::
+   Use this section to describe the library architecture and implementation.
+
+Supported features
+==================
+
+.. note::
+   Use this section to describe the features supported by the library.
+
+Supported backends*
+===================
+
+.. note::
+   Use this section to describe the backends supported by the library, if applicable.
+   This is optional.
+
+Requirements
+************
+
+.. note::
+   Use this section to list the requirements needed to use the library.
+   This is optional.
+
+Library files
+*************
+
+.. note::
+   Use this section to indicate the location of the library files.
+
+This library can be found under :file:`path_to_library_files` in the |NCS| folder structure.
+
+Configuration*
+**************
+
+.. note::
+   Use this section to list any library configuration option available, if applicable.
+   This is optional.
+
+Shell commands list*
+====================
+
+.. note::
+   Use this section to list any library shell command available, if applicable.
+   This is optional.
+
+Usage
+*****
+
+.. note::
+   Use this section to explain how to use the library.
+
+Samples using the library*
+**************************
+
+.. note::
+   Use this section to explain how to list all the NCS samples using this library, if present.
+   This is optional.
+
+The following |NCS| samples use this library:
+
+* Sample 1
+* Sample 2
+* Sample 3
+
+Application integration*
+************************
+
+.. note::
+   Use this section to explain how to integrate the library in a custom application.
+   This is optional.
+
+Additional information*
+***********************
+
+.. note::
+   Use this section to describe any additional information relevant to the user.
+   This is optional.
+
+Limitations*
+************
+
+.. note::
+   Use this section to describe any eventual limitation of the library, if present.
+   This is optional
+
+Dependencies
+************
+
+.. note::
+   Use this section to list all the dependencies of this library, if applicable.
+
+API documentation
+*****************
+
+.. note::
+   Use this section to call the API documentation as follows:
+
+   .. code-block::
+
+      | Header file: :file:`*indicate_the_path*`
+      | Source files: :file:`*indicate_the_path*`
+
+      .. doxygengroup:: *indicate_the_doxygroup*
+         :project: *indicate_the_project_name*
+         :members:
```

Comment:
Since this is required, shouldn't it be outside of the note?
Or is it in the note and code block because it'll otherwise throw an error?

Reply:
Exactly, It is the code needed to show the API docs

Reply:
Right, the code block has eaten it. Will fix it